spec/requests/validations_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ValidationsController do
+  let(:schooling) { create(:schooling) }
+  let(:student) { schooling.student }
+  let(:user) { create(:user, :director, :with_selected_establishment, establishment: schooling.classe.establishment) }
+  let(:classe) { schooling.classe }
+  let(:pfmp) { create(:pfmp, :completed, schooling: schooling) }
+
+  before { sign_in(user) }
+
+  describe "GET #index" do
+    it "renders the :index template" do
+      get validations_path
+
+      expect(response).to render_template :index
+    end
+  end
+
+  describe "GET #show" do
+    before { get validation_class_path(classe) }
+
+    it "assigns the requested classe to @classe" do
+      expect(assigns(:classe)).to eq classe
+    end
+
+    it "renders the :show template" do
+      expect(response).to render_template :show
+    end
+  end
+
+  describe "POST #validate" do
+    context "with valid attributes" do
+      it "validates selected pfmps" do
+        post validation_class_path(classe), params: { validation: { pfmp_ids: [pfmp.id] }, confirmed_director: "1" }
+
+        expect(pfmp.reload.current_state).to eq("validated")
```

Comment:
tu peux utiliser le `change` comme ci-dessous

Reply:
Je pense qu'il veut dire que tu peux faire `expect {pfmp.reload.current_state}.to change_to("validated")` ou un truc comme ça, pour t'assurer que son statut vaut validated, mais aussi qu'il a été modifié (qu'il est pas juste resté validated)

Reply:
Ah mais oui on peut check change from to, bien vu